test/functional/cmdLineTests/modularityddrtests/modularityddrtests.xml

```diff
@@ -372,5 +372,184 @@
   <output regex="no" type="failure">DDRInteractiveCommandException</output>
 
  </test>
- 
- </suite>
\ No newline at end of file
+
+
+
+ <!-- Test !findmodules and all options -->
+ <test id="Run !findmodules">
+  <exec command="sh" capture="LOGNAME" platforms="zos.*" >
```

Comment:
Please indent consistently: one tab (or a fixed number of spaces) per level. I'll let you decide whether you want to 'fix' the rest of the file.

Also, can you remove the extra blank lines above (one should be sufficient).

Reply:
In the future the rest of the tests will be removed, leaving just new tests. As a result, I have only fixed indentation on new tests.